src/main/java/org/jenkinsci/plugins/ghprb/GhprbCause.java

```diff
@@ -30,13 +30,13 @@
 
     private final URL url;
 
-    private final GHUser triggerSender;
+    private final transient GHUser triggerSender;
```

Comment:
Note: as mentioned in JIRA, this will still cause problems for loading _historical_ builds (predating this fix and Jenkins 2.102). Probably solvable simply by renaming the fields—but untested.